src/status_im/subs/wallet/wallet_connect.cljs

```diff
@@ -37,14 +38,24 @@
 (rf/reg-sub
  :wallet-connect/current-request-dapp
  :<- [:wallet-connect/current-request]
- :<- [:wallet-connect/pairings]
- (fn [[request pairings]]
+ :<- [:wallet-connect/sessions]
+ (fn [[request sessions]]
    (let [dapp-url (get-in request [:event :verifyContext :verified :origin])]
-     (->> pairings
-          (filter (fn [pairing]
-                    (= dapp-url (get-in pairing [:peerMetadata :url]))))
+     (->> sessions
+          (filter (fn [session]
+                    (= dapp-url (get session :url))))
           (first)))))
 
+(rf/reg-sub
+ :wallet-connect/sessions-for-current-account
+ :<- [:wallet-connect/sessions]
+ :<- [:wallet/current-viewing-account]
+ (fn [[sessions {:keys [address]}]]
```

Comment:
The `wallet/current-viewing-account-address` sub would be best here as we need only the address. 

The `wallet/current-viewing-account` might trigger unwanted re-renders if the account is updated from the signal.